src/test/libslic3r/test_support_material.cpp

```diff
@@ -1,20 +1,20 @@
 #include <catch.hpp>
-#include <libslic3r/IO.hpp>
-#include <libslic3r/GCodeReader.hpp>
+#include "GCodeReader.hpp"
+#include "test_data.hpp"
 
-#include "libslic3r.h"
-#include "TriangleMesh.hpp"
-#include "Model.hpp"
-#include "SupportMaterial.hpp"
+// For IDE suggestions
+#include "../../../build/external/Catch/include/catch.hpp"
```

Comment:
You probably should just not stage these lines in the future. 

This line will crash if someone builds out of tree.